### ISSUE TITLE
Modal:  won't put attribute on outermost element

### ISSUE THREAD

**Issue by knod** [Semantic-Org/Semantic-UI-React#3195]
## Bug Report
I'm not sure if this is a bug or if this is supposed to be the expected behavior. Sorry if this is in the wrong place. Also, I couldn't find a duplicate issue, but I wasn't sure how to search for it. Thanks for taking a look!

### Steps
1. Create a modal component with an id attribute
2. Inspect it with the inspector tools

### Expected Result
The id would be on the outermost wrapping div - the one that currently has the classes 'ui page modals...' If I create any component with an id, I'd expect the id to be on the outermost element created, especially since CSS can't select parents. It's especially a problem if there are multiple modals open at the same time.

I'd expect to be able to do styles such as `#myModal ~ div { user-select: none; }` so that when people that need to use keyboard input, such as people with disabilities, can tab through the interactive elements in the modal without tabbing through ~the page behind it too~. [My bad, you can't select previous siblings, so that particular issue wouldn't be solved by this. It still prevents other ways of using CSS on the element and its successor siblings.]

### Actual Result
The next element in, the div with the `modal` class is the one that gets the id, preventing CSS rules from reaching the parent and its siblings.

### Version
^0.75.1

### Testcase
https://codesandbox.io/s/pkwwq712qm

**Comment by knod** [comment]
But no one has responded to this yet.

**Comment by layershifter** [comment]
Will be implemented in #3195. API will be:

```js
<Modal 
  dimmer={{ className: 'custom-classname', inverted: true }}
/>
```
